cpu/atmega_common/periph/spi.c

```diff
@@ -92,7 +92,7 @@ void spi_release(spi_t bus)
 void spi_transfer_bytes(spi_t bus, spi_cs_t cs, bool cont,
                         const void *out, void *in, size_t len)
 {
-    uint8_t *out_buf = (uint8_t *)out;
+    const uint8_t *out_buf = (const uint8_t *)out;
```

Comment:
sure? Because below it is:

```
uint8_t tmp = (out_buf) ? out_buf[i] : 0;
```

assigning `void` to `uint8_t` without cast is possible?